config.go

```diff
@@ -13,7 +13,9 @@ type Config struct {
 		Server        string
 		Port          int
 		Nick          string
+		Password      string
 		Channel       string
+		SendMUserName bool
```

Comment:
A bool is false if uninitialized, so the keep backwards compatibility with existing configs, better make this the inverse. Like DisableMmUserName